api/src/executables/getWireguardCredentialsLocal.ts

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env node
+
+// This script must return the MASTER_ADMIN credentials formated
+// as a nice message in the terminal + a QR code
+//
+// It will be run as a separate process, maybe in a different container
+
+import { renderQrCode } from "../renderQrCode";
+import { params } from "../params";
+import { createLocalConfigFile } from "../createLocalConfigFile";
+
+// If user does CTRL + C, exit process
+process.on("SIGINT", () => process.exit(128));
+
+(async function (): Promise<void> {
+  try {
+    console.log(
+      `Fetching local Wireguard credentials. It may take some time; use CTRL + C to stop`,
+    );
+
+    const wireguardLocalCredentials = await createLocalConfigFile(params.MASTER_ADMIN);
+
+    // If rendering the QR fails, show the error and continue, the raw URL is consumable
+    console.log(`
+
+${await renderQrCode(wireguardLocalCredentials).catch((e) => e.stack)}`);
```

Comment:
Should we just let the error by catched by the outter try / catch?

Reply:
Also, do we want to render the QR or the config file itself?

Reply:
1. I think that would be useful to render the log in case of error in the console

2. We would render the config file for `dappnode_wireguard`
For qr codes there are defined 2 other commands for local and remote:
    - `dappnode_wireguard_local`
    - `dappnode_wireguard_remote`

More info on: https://github.com/dappnode/DAppNode_Installer/pull/150